src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/Link.js

```diff
@@ -115,7 +115,7 @@ class Link extends Component<Props> {
                 target: enableTarget ? target : undefined,
                 anchor: enableAnchor ? anchor : undefined,
                 href,
-                title: enableTitle ? title : undefined,
+                title: enableTitle || (title != undefined && title.length) ? title : undefined,
```

Comment:
I think the logic before here is correctly the title should never be saved here when enableTitle is set to false (onChange is saving it). I think the problem is that it need to hold the title here internal something like:

```js
this.overlayTitle = title;
```

because if we are calling the title with `onChange` and later change the title of the page it will not show the new title of that page then.